SCClassLibrary/Common/Streams/History.sc

```diff
@@ -464,27 +480,17 @@ History { 		// adc 2006, Birmingham; rewrite 2007.
 
 	makeWin { |where, numItems=8|
 		var gui = HistoryGui(this, numItems);
-		if (where.notNil) { gui.moveTo(where.x, where.y); };
+		if (where.notNil) { gui.moveTo(where.x, where.y) };
 		^gui
 	}
 
 	*document { current.document }
 
-	document { arg title="";	// platform dependent ...
-		var docTitle;
-		Platform.case(
-			\windows, {
-				// not sure this works in 3.7.0?
-				this.storyString.newTextWindow("History_documented");
-			},
-			{
-				//
-				docTitle = title ++ Date.getDate.format("%Y-%m-%e-%Hh%M-History");
-				Document.new(docTitle, this.storyString)
-				// path not working yet
-				//	.path_(docTitle); // don't lose title.
-			}
-		)
+	document { |title = ""|
+		var docTitle = title ++ Date.getDate.format("%Y-%m-%e-%Hh%M-History");
```

Comment:
ahh, this is a problem. Windows' `strtime` function doesn't allow `%e` (day of month w/o leading zeroes). Two options for a fix (local fix anyway): use `%d` (day of month, _with_ leading zero), or use `%e` on unix and `%#d` on Windows (just tested, this already produces the correct result).

Perhaps later we should modify Date.format to internally convert %e to %#d on windows. As it stands, the documentation doesn't make this point clear (although it does make a note it depends on your system's strtime implementation).